### ISSUE TITLE
serving from production build locally

### ISSUE THREAD

**Issue by Lutando** [maxmantz/redux-oidc#28]
So I am using this in my app and love it. I do have a problem. When I run the `dist` of the example in `redux-oidc-example` the application can't seems to parse the hash/url fragment returned by google because the webserver reports a 404 on /callback and basically the callback component never gets mounted.

(i am using npms `http-server` to serve the spa using your) I am getting a similar problem against another oidc provider. 

It seems like I need to somehow tell my webserver that when the provider calls /callback it should pass that to the app. Do you know how this would be done in express?

basically how do i expose my apps callback? i see your prod server doesn't have anything that might give me a clue about this. TIA

**Comment by Lutando** [comment]
Figured it out
